knative-operator/pkg/controller/knativeserving/knativeserving_controller.go

```diff
@@ -151,14 +152,19 @@ func (r *ReconcileKnativeServing) deleteVirtualService(instance *servingv1alpha1
 	list := &v1alpha3.VirtualServiceList{}
 	ctx := context.TODO()
 	if err := r.client.List(ctx, listOpts, list); err != nil {
+		if meta.IsNoMatchError(err) {
+			log.Info("failed to list VirtualServices")
+			// VirtualService CRD is not installed.
```

Comment:
Why log this? This is going to be super confusing. Let's just return nil.

Reply:
I misunderstood the last comment :cold_sweat: Sorry! Removed it now.